public_website/__tests__/about.test.tsx

```diff
@@ -22,12 +23,40 @@ describe('About page', () => {
   })
 
   it('should render the page', async () => {
-    const { container } = render(<About />)
+    const { container } = render(<About restaurants={[]} />)
 
     await waitFor(() => {
       expect(screen.queryByText('Chargement...')).toBeFalsy()
     })
 
     expect(container).toMatchSnapshot()
   })
+
+  it('should render the page with the response of the server', async () => {
+    process.env = { ...process.env, ID_TOKEN: 'your_dummy_token' }
+
+    const { props } = await getStaticProps()
+    render(<About {...props} />)
+    expect(screen.queryByText('Tacos de Lyon')).toBeTruthy()
+  })
+
+  it('should render the page with restaurants', async () => {
+    const { container } = render(<About restaurants={restaurantDataFixtures} />)
+
+    await waitFor(() => {
+      expect(screen.queryByText('Chargement...')).toBeFalsy()
+    })
```

Comment:
Ce bloc de code est répété plusieurs fois et n'est pas super explicite

Je suggère d'extraire une fonction du style


```suggestion
    await waitDataToBeLoadedAndRenderedFor()
```